### PR TITLE
Add Sha1 calculator by name, version, arch, branch and dependency type. + Fix Methods Description

### PR DESCRIPTION
**Pull request by Shereind**

### CODE REVIEW

File: wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java [hunk 1]
```diff
@@ -312,8 +333,8 @@ public String calculateSha1ByNameVersionArchitectureAndType(String name, String
     /**
      * Calculates SHA-1 for library by name, version and dependencyType
      *
-     * @param name of library
-     * @param version of library
+     * @param name           of library
```
Comment (annarozin) [inline wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java:336]: why do we need these spaces?

File: wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java [hunk 2]
```diff
@@ -251,55 +254,73 @@ public String calculateByteArrayHash(byte[] byteArray, HashAlgorithm algorithm)
...
 
         return calculateByteArraySHA1(sha1ToCalc.getBytes(StandardCharsets.UTF_8));
     }
 
     /**
      * Calculates SHA-1 for library by name, version, architecture, release and dependencyType
      *
-     * @param name of library
-     * @param version of library
+     * @param name           of library
```
Comment (annarozin) [inline wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java:283]: revert to the version without these spaces

**Review by annarozin — COMMENTED** [review]
please review my question
In addition, the comment for the pull request is not accurate, the release is missing in it

File: wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java (hunk 1 above)
Comment (Shereind) [reply wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java:336]: This what happens when I press ctrl+shift+L -> Its to fix file format 
Do you want to leave it like this or revert it?

**Comment by Shereind** [comment]
I only fixed the description for calculating additional sha1 with release

File: wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java (hunk 1 above)
Comment (annarozin) [reply wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java:336]: revert please

**Review by annarozin — APPROVED** [review]
approved

**Pull request merged**